From a9d6d7b01836eff6e896bb5e72f0868217b88226 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Mon, 20 Mar 2017 11:41:37 +0000
Subject: [PATCH 158/214] Update project name in the footer

 * Make it consistent with the name shown on the wiki

Change-Id: Ia224c1038f9e7e03948b4708fea40e11a3aafc24
---
 _layouts/default.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_layouts/default.html b/_layouts/default.html
index 0fc6ae9..b701881 100644
--- a/_layouts/default.html
+++ b/_layouts/default.html
@@ -60,7 +60,7 @@
           <div class="col-sm-6">
             <img class="footer-img" src="{{ site.baseurl }}/images/logo-3.png" alt="LineageOS Logo">
             <p class="text-muted">
-              <b>&copy; 2016 - {{ site.time | date: '%Y' }} LineageOS Project </b><br />
+              <b>&copy; 2016 - {{ site.time | date: '%Y' }} The LineageOS Project </b><br />
               We are currently not accepting monetary donations. This message will be removed when there is an official method to help fund the project.
             </p>
           </div>
-- 
2.17.1

